.travis.yml

```diff
@@ -5,8 +5,10 @@ services:
 
 before_script:
   - docker-compose -f docker-compose.yml -f docker-compose.travis.yml up -d
+  - docker-compose ps
   - docker-compose exec web sh -c "apk add --no-cache git"
-  - docker-compose exec web sh -c "pipenv install -d tox"
```

Comment:
We should also keep this `-d` flag, that way pipenv doesn't go through the trouble unnecessarily of creating a local virtualenv when it's in the docker container.